lib/class-wp-rest-widgets-controller.php

```diff
@@ -116,7 +92,7 @@ public function register_routes() {
 			),
 			'schema' => array( $this, 'get_public_item_schema' ),
 		) );
-		register_rest_route( $this->namespace, '/' . $this->rest_base .'/types/(?P<type>[\w-]+)', array(
+		register_rest_route( $this->namespace, '/' . $this->rest_base_types . '/(?P<type>[\w-]+)', array(
```

Comment:
@BE-Webdesign take a look at https://github.com/WP-API/wp-api-menus-widgets-endpoints/issues/12 and specifically https://github.com/WP-API/wp-api-menus-widgets-endpoints/issues/12#issuecomment-177399817 where we discussed forward/backward compatibility regarding Widget IDs.